### PR TITLE
Kelsonic 7106 yr mobile

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/7106

This PR adds both the functionality and the styling to match the [mobile design spec](https://vsateams.invisionapp.com/share/36W0CAZHJ2U#/screens/405289290) for YR.

## Testing done
Updated unit tests and added a couple.

#### Desktop Page 1
localhost_3001_education_yellow-ribbon-participating-schools_

#### Desktop Page 2
localhost_3001_education_yellow-ribbon-participating-schools_ (1)

#### Mobile Page 1
localhost_3001_education_yellow-ribbon-participating-schools_(Galaxy S5)

#### Mobile Page 2 (collapsed)
localhost_3001_education_yellow-ribbon-participating-schools_(Galaxy S5) (1)

## Acceptance criteria
- [x] Update YR pages to match mobile design spec.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]

**Comment by rianfowler** [comment]
looks like it already is- I was confused. I thought the review group was added as a required approver.

File: src/applications/yellow-ribbon/components/SearchResultsPage/index.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        {/* Search Form */}
+        <div className="vads-l-col--12 medium-screen:vads-l-col--3">
+          {/* Toggle Mobile Form */}
+          <button
+            className="usa-button-secondary usa-button-big medium-screen:vads-u-display--none vads-u-font-size--md"
+            onClick={toggleShowMobileForm}
+            type="button"
+          >
+            Change search criteria{' '}
+            <i className="fa fa-chevron-down vads-u-padding-left--0p5" />
```
[the remark this answers is not in the document]
Comment (bkjohnson) [reply src/applications/yellow-ribbon/components/SearchResultsPage/index.js:55]: This icon doesn't seem to be semantic, so it's safe to ignore the bot here.

File: src/applications/yellow-ribbon/components/FindYellowRibbonPage/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import manifest from '../../manifest.json';
+
+export const FindYellowRibbonPage = () => (
+  <>
+    {/* Breadcrumbs */}
+    <Breadcrumbs className="vads-u-padding--1p5 medium-screen:vads-u-pading--0">
+      <a href="/">Home</a>
+      <a href="/education/">Education and training</a>
+      <a href={manifest.rootUrl}>Find a Yellow Ribbon school</a>
+    </Breadcrumbs>
```
Comment (ncksllvn) [inline src/applications/yellow-ribbon/components/FindYellowRibbonPage/index.js:16]: You could possibly save a little code by a shared breadcrumbs component -

```jsx
export default function AppBreadcrumbs({ children }) {
    return (
     <Breadcrumbs className="vads-u-padding--1p5 medium-screen:vads-u-pading--0">
      <a href="/">Home</a>
      <a href="/education/">Education and training</a>
      <a href={manifest.rootUrl}>Find a Yellow Ribbon school</a>
      {children}
    </Breadcrumbs>
    );
}
```

**Review by ncksllvn — APPROVED** [review]
LGTM! Well done 👏

**Pull request merged**